crypto/signer/test/faulty_signer.go

```diff
@@ -1,14 +1,15 @@
 package test
 
 import (
+	"context"
 	"github.com/orbs-network/go-mock"
 )
 
 type FaultySigner struct {
```

Comment:
why "faulty"? isn't this simply a mock?

Reply:
also, you only use it in one test, so maybe move it to that test file? don't prematurely generalize